ppr-ui/src/views/newMhrRegistration/HomeOwners.vue

```diff
@@ -329,6 +368,13 @@ export default defineComponent({
       }
     )
 
+    onMounted(() => {
```

Comment:
We can update to onBeforeMount() to hide owners before they show up in the table.